src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -124,10 +130,16 @@ public static class TextCatalog
             => new TextNormalizingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(),
                 outputColumnName, inputColumnName, caseMode, keepDiacritics, keepPunctuations, keepNumbers);
 
-        /// <include file='doc.xml' path='doc/members/member[@name="WordEmbeddings"]/*' />
```

Comment:
>'doc.xml' [](start = 26, length = 9)

if doc.xml is no longer used, please remove the file

Reply:
The file is still used by the trasnforms that Ivan, and senja are workign on . I would rather remove it at the end, and not modify it to avoid conflicts.

---
In reply to: [277032552](https://github.com/dotnet/machinelearning/pull/3418#discussion_r277032552) [](ancestors = 277032552)